cypress/support/pages/order.pageObject.js

```diff
@@ -0,0 +1,37 @@
+import PageObject from '../PageObject';
+
+class OrderPageObject extends PageObject {
+  fillOrderForm() {
+    cy.get('#name').type('Test Name'); // Змініть на реальні дані
+    cy.get('#country').type('Test Country');
+    cy.get('#city').type('Test City');
+    cy.get('#card').type('1234567890123456');
+    cy.get('#month').type('12');
+    cy.get('#year').type('2025');
+  }
+
+  get purchaseButton() {
+    return cy.contains('Purchase');
+  }
+
+  clickPurchase() {
+    this.purchaseButton.click();
+  }
+
+  assertOrderDetails() {
+    
+    cy.get('.modal-body').should('contain', 'Test Name');
+    cy.get('.modal-body').should('contain', 'Test Country');
+    cy.get('.modal-body').should('contain', 'Test City');
```

Comment:
The assertions in `assertOrderDetails` method are currently using hardcoded values ('Test Name', 'Test Country', 'Test City'). Consider using dynamic values or parameters to make the test more flexible and applicable to different scenarios.